packages/components/select-v2/src/useSelect.ts

```diff
@@ -268,7 +268,33 @@ const useSelect = (props: ISelectV2Props, emit: SelectEmitFn) => {
   )
 
   const calculatePopperSize = () => {
-    popperSize.value = selectRef.value?.offsetWidth || 200
+    if (typeof props.fitInputWidth === 'number') {
```

Comment:
Use built-in packaged functions.
```suggestion
    if (isNumber(props.fitInputWidth)) {
```